src/thor/costmatrix.cc

```diff
@@ -53,8 +53,8 @@ CostMatrix::CostMatrix(const boost::property_tree::ptree& config)
       max_reserved_labels_count_(config.get<uint32_t>("max_reserved_labels_count_bidir_dijkstras",
                                                       kInitialEdgeLabelCountBidirDijkstra)),
       max_reserved_locations_count_(
-          config.get<uint32_t>("max_reserved_locations_costmatrix", kMaxLocationReservation)),
-      check_reverse_connections_(config.get<bool>("costmatrix_check_reverse_connection", false)),
+          config.get<uint32_t>("costmatrix.max_reserved_locations", kMaxLocationReservation)),
```

Comment:
I moved these from `thor` to the new `thor.costmatrix` section. Right now this would silently ignore values at the old location, which sucks. So the question is: do we warn? I would like to avoid crashing the service the way we do in service limits. I personally hate when that happens to me :smile:

Reply:
Another thing we could do is release 3.5.2 and pin a warning at the top of the release notes.